packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -804,23 +806,50 @@ class RawGestureDetector extends StatefulWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// Describes the semantics notations that should be added to the underlying
+  /// render object [RenderSemanticsGestureHandler].
+  ///
+  /// It has no effect if [excludeFromSemantics] is true.
+  ///
+  /// When [semantics] is null, [RawGestureDetector] will fall back to a
+  /// default delegate which checks if the detector owns certain gesture
+  /// recognizers and calls their callbacks if they exist.
+  ///
+  ///  * During a semantic tap, it looks for [TapGestureRecognizer] and calls
+  ///    `onTapDown`, `onTapUp`, and `onTap`.
```

Comment:
"it looks for... and calls..." makes it sound like it doesn't do anything with the result of looking for it. it just does it and calls the callbacks regardless of the result.

Reply:
Fixed, see https://github.com/flutter/flutter/pull/33936/commits/9111a12eb6d73627c5079371672b1e471dd8a477